pandas/core/nanops.py

```diff
@@ -1238,7 +1238,7 @@ def _maybe_null_out(
     result: np.ndarray,
     axis: Optional[int],
     mask: Optional[np.ndarray],
-    shape: Tuple,
+    shape: Tuple[int],
```

Comment:
does `Tuple[int]` describe a 1-tuple, or any-length-tuple?

Reply:
Ah, yes, apparently that needs to be `Tuple[int, ...]` for a variable length tuple